scripts/add_sa_role.sh

```diff
@@ -1,16 +1,13 @@
 #!/bin/bash
 
-set -x
+set -ex
 
 ORG_ID=$(gcloud organizations list --format 'value(ID)')
-TIMESTAMP=$(date +%s)
-DELTA=3000
-TIMESTAMP_PLUS_EXPIRY=$((TIMESTAMP + DELTA))
-EXPIRY=$(printf '%(%Y-%m-%dT%H:%M:%SZ)T' $TIMESTAMP_PLUS_EXPIRY)
+EXPIRY=$(python3 -c 'from datetime import datetime, timedelta; now = datetime.utcnow(); expires = now + timedelta(hours=1); print(expires.strftime("%Y-%m-%dT%H:%M:%SZ"))')
 SA_PROJECT=$(gcloud config get-value project)
 SERVICE_ACCOUNT="terraform@$SA_PROJECT.iam.gserviceaccount.com"
 
 gcloud organizations add-iam-policy-binding "$ORG_ID" \
   --member="serviceAccount:$SERVICE_ACCOUNT" \
   --role "organizations/$ORG_ID/roles/terraformServiceAccount" \
-  --condition="expression=request.time < timestamp('$EXPIRY'),title=Expiry: $EXPIRY"
+  --condition="expression=request.time < timestamp('$EXPIRY'),title=Expiry: $EXPIRY"
```

Comment:
Please fix the newline issue - you should be able to configure your text editor to automatically add a newline at the end of files 👍

Reply:
Done!